clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -4050,7 +4050,7 @@ void TokenAnnotator::calculateFormattingInformation(AnnotatedLine &Line) const {
                              ChildSize + Current->SpacesRequiredBefore;
     }
 
-    if (Current->is(TT_CtorInitializerColon))
+    if (Current->isOneOf(TT_CtorInitializerColon, TT_LambdaLSquare))
```

Comment:
is this bit tested or unrelated?

Reply:
This is needed so that lambda definitions are not treated as `InFunctionDecl`, which now is used on line 4214 to filter out function trailing return arrows. We already got tests (line 22818 and 25446 in FormatTest.cpp) that would fail without it.